server/apps/users/helpers.py

```diff
@@ -53,3 +54,35 @@ def get_user_profile(user):
     except UserProfile.DoesNotExist:
         uo = None
     return uo
+
+def delete_user(user, delete_oh_data):
+    """
+    Deletes the user and all data associated with it.
+
+    Args:
+        delete_oh_data: True if stories on OpenHumans should also be deleted
+
+    Returns:
+        None
+    """
+    ohmember = user.openhumansmember
+
+    # Delete the stories from the AutSPACEs database
+    PublicExperience.objects.filter(
+        open_humans_member=ohmember
+    ).delete()
+
+    # Delete the stories from the OpenHumans database
+    if delete_oh_data:
+        ohmember.delete_all_files()
+
+    # Delete the user's profile
+    UserProfile.objects.filter(
+        user=user
+    ).delete()
+
+    # Delete the actual user
+    User.objects.filter(
+        id=user.id
+    ).delete()
```

Comment:
I think all of these except for the `oh_member.delete_all_files()` are not necessary actually, as I'm 99% sure that all of our models do `on_delete=models.CASCADE`, which means that a deletion of the main `User` will also delete all the other objects!

Reply:
Great; and the unit tests seem to confirm it. I've pushed a simplifed version.